lib/Models/ImageryLayerCatalogItem.js

```diff
@@ -452,7 +433,7 @@ ImageryLayerCatalogItem.prototype._enable = function(layerIndex) {
     if (defined(this.intervals) && this.intervals.length > 0) {
         isTimeDynamic = true;
 
-        var clock = this.clockForDisplay;
+        var clock = this.terria.clock;
```

Comment:
Shouldn't this be `this.clock` instead of `this.terria.clock`?

Reply:
Thanks for picking this up. Offline we sorted out that multiplier should be terria.clock and currenttime should be this.clock.

This and another issue that I discovered in the process are addressed in : bbf0363bb1a3fc49f0165c90579cf47d48571628, ca085e492a4de6d320ee227f82ad830fc0ae9d27, 49403388f258d8c35c86004773d60cdd5366767e.